apps/app/src/views/MyReservations.vue

```diff
@@ -6,18 +6,25 @@
       :items="reservations"
       :items-length="reservations.length"
       :loading="isLoading">
+      <template #[`item.actions`]="{ item }">
+        <ReservationDelete
```

Comment:
I am not sure if `ReservationDelete` reflects its purpose the best, it is not clear is it a button, card, or something else. Can we maybe find a better name for this?